### ISSUE TITLE
[DEV] Build `train.py` and the rest of training pipeline

### ISSUE THREAD

**Issue by noisrucer** [noisrucer/sign-language-detection#16]
## Title
Build training pipeline

## Description
YOLOv2 model, dataset, dataloader, loss function are all set. Time to train!

**Comment by noisrucer** [comment]
* Finished a rough draft of `train.py`
* Finished the whole pipeline
* Need to better organize folder structures after testing
